### PR TITLE
Language updates with Andy

### PR DESCRIPTION
**Pull request by marksou**

### CODE REVIEW

File: tyk-docs/content/basic-config-and-security/security/Authentication & Authorization/auth-code-grant-type.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ curl -X POST \
 
 #### Response
 
-Response generates a 307 Temporary Redirect to the Oauth client redirect URI. It is expected that this location will be capable of authenticating the user then using the data forwarded to it as part of the redirect to request an authorization code.
+Response generates a 307 Temporary Redirect to the Oauth client redirect URI. It is expected that this location will be capable of authenticating the user then using the data forwarded to it as part of the redirect to request an `authorization` code.
 
 ## Request an authorization code
```
Comment (andyo-tyk) [inline tyk-docs/content/basic-config-and-security/security/Authentication & Authorization/auth-code-grant-type.md:47]: Should this be authorisation, or in code format?

File: tyk-docs/content/basic-config-and-security/security/Authentication & Authorization/json-web-tokens/split-token.md [hunk 2]
```diff
@@ -57,19 +57,18 @@ Signature
...
+So back to Tyk, the API Gateway is perfectly positioned to act as a broker between the client and the authorisation server.  It can accept requests for new access tokens, given a client id and secret, and exchange that for an access token with the authorisation server. Then, it will break apart the JWT and return only the signature portion back to the client.  It then stores the rest of the JWT internally.
 
 This means that the client can then simply use the signature as an access token and Tyk can validate the token as if it was a normal API Key.  Then it will pull the JWT out of cache and inject claims into the request or even rebuild the access token, which is safe.
 
 
 **Let’s get started IdP**
 
-Inside Tyk, let’s create a virtual endpoint or API, listening to the path /token. This virtual endpoint is responsible for receiving the auth request from the client, and acting as an identity broker with the authorization server.
+Inside Tyk, create a virtual endpoint or API, listening to the path `/token`. This virtual endpoint is responsible for receiving the auth request from the client, and acting as an identity broker with the authorization server.
```
Comment (andyo-tyk) [inline tyk-docs/content/basic-config-and-security/security/Authentication & Authorization/json-web-tokens/split-token.md]: This should be authorisation for consistency

File: tyk-docs/content/basic-config-and-security/security/Authentication & Authorization/oauth-2-0.md [hunk 3]
```diff
@@ -38,11 +38,11 @@ The Tyk OAuth flow is described in the following sections.
 
 ### Authorisation token flow (e.g. server-side web apps)
 
-1.  Resource owner registers a new Client ID with Tyk
-2.  Client makes a request on behalf of an end user to `/oauth/authorize/` on your Tyk instance `listen_path`
-3.  Tyk will check the OAuth authorize request for validity (i.e. Does the Client ID exist and is the request properly formed to the OAuth 2.0 standard)
+1.  Resource owner registers a new Client ID with Tyk.
+2.  Client makes a request on behalf of an end user to `/oauth/authorize/` on your Tyk instance `listen_path`.
+3.  Tyk will check the OAuth authorize request for validity (i.e. Does the Client ID exist and is the request properly formed to the OAuth 2.0 standard).
```
Comment (andyo-tyk) [inline tyk-docs/content/basic-config-and-security/security/Authentication & Authorization/oauth-2-0.md:43]: Should this authorize be in `code` format?

File: tyk-docs/content/basic-config-and-security/security/dashboard/dashboard-api-security.md [hunk 4]
```diff
@@ -25,4 +25,4 @@ See the [User API Roles](/docs/basic-config-and-security/security/dashboard/user
 
 ## Dashboard API Access
 
-The Dashboard API is secured using an “Authorization” header that must be added to each request that is made. The Authorization key can be found in the User details view of a user.
\ No newline at end of file
+The Dashboard API is secured using an `Authorization` header that must be added to each request that is made. The **Tyk Dashboard API Access Credentials** Authorization key can be found at the bottom of the **Edit User** section for a user.
```
Comment (andyo-tyk) [inline tyk-docs/content/basic-config-and-security/security/dashboard/dashboard-api-security.md]: Should the second Authorization also be in code format?

File: tyk-docs/content/plugins/auth-plugins/id-extractor.md [hunk 5]
```diff
@@ -19,11 +19,11 @@ When a custom authentication mechanism is used, every API call triggers a call t
 
 The ID extractor works for all rich plugins: gRPC-based plugins, Python and Lua.
 
-## <a name="when"></a>When to use the ID Extractor?
+## When to use the ID Extractor?
 
-The main idea of the ID extractor is to reduce the number of calls made to your plugin and cache the API keys that have been already authorized by your authentication mechanism.This means that after a successful authentication event, subsequent calls will be handled by the Tyk Gateway and its Redis cache, resulting in a performance similar to the built-in authentication mechanisms that Tyk provides.
+The main idea of the ID extractor is to reduce the number of calls made to your plugin and cache the API keys that have been already authorised by your authentication mechanism.This means that after a successful authentication event, subsequent calls will be handled by the Tyk Gateway and its Redis cache, resulting in a performance similar to the built-in authentication mechanisms that Tyk provides.
```
Comment (andyo-tyk) [inline tyk-docs/content/plugins/auth-plugins/id-extractor.md]: Missing space after full stop: mechanism.This

Comment (andyo-tyk) [reply tyk-docs/content/plugins/auth-plugins/id-extractor.md]: Also, on line 31 below, there's a missed z in "you initialize the session object"

File: tyk-docs/content/tyk-on-prem/installation/debian-ubuntu/analytics-pump.md [hunk 6]
```diff
@@ -33,7 +33,7 @@ $ git clone https://github.com/TykTechnologies/tyk-ansible
 $ cd tyk-ansible
 ```
 
-3. Run initalization script to initialize environment
+3. Run initalisation script to initialise environment
```
Comment (andyo-tyk) [inline tyk-docs/content/tyk-on-prem/installation/debian-ubuntu/analytics-pump.md]: Typo: _initalisation_

File: tyk-docs/content/tyk-on-prem/installation/debian-ubuntu/dashboard.md [hunk 7]
```diff
@@ -33,7 +33,7 @@ $ git clone https://github.com/TykTechnologies/tyk-ansible
 $ cd tyk-ansible
 ```
 
-3. Run initalization script to initialize environment
+3. Run initalisation script to initialise environment
```
Comment (andyo-tyk) [inline tyk-docs/content/tyk-on-prem/installation/debian-ubuntu/dashboard.md]: Typo: _initalisation_

File: tyk-docs/content/tyk-on-prem/installation/debian-ubuntu/debian-ubuntu.md [hunk 8]
```diff
@@ -35,7 +35,7 @@ $ git clone https://github.com/TykTechnologies/tyk-ansible
 $ cd tyk-ansible
 ```
 
-3. Run initalization script to initialize environment
+3. Run initalisation script to initialise environment
```
Comment (andyo-tyk) [inline tyk-docs/content/tyk-on-prem/installation/debian-ubuntu/debian-ubuntu.md]: Typo: _initalisation_

File: tyk-docs/content/tyk-on-prem/installation/debian-ubuntu/gateway.md [hunk 9]
```diff
@@ -33,7 +33,7 @@ $ git clone https://github.com/TykTechnologies/tyk-ansible
 $ cd tyk-ansible
 ```
 
-3. Run initalization script to initialize environment
+3. Run initalisation script to initialise environment
```
Comment (andyo-tyk) [inline tyk-docs/content/tyk-on-prem/installation/debian-ubuntu/gateway.md]: Typo: _initalisation_

File: tyk-docs/content/tyk-on-prem/installation/install-tyk-ansible.md [hunk 10]
```diff
@@ -27,7 +27,7 @@ $ git clone https://github.com/TykTechnologies/tyk-ansible
 $ cd tyk-ansible
 ```
 
-3. Run initalization script to initialize environment
+3. Run initalisation script to initialise environment
```
Comment (andyo-tyk) [inline tyk-docs/content/tyk-on-prem/installation/install-tyk-ansible.md]: Typo: _initalisation_

File: tyk-docs/content/tyk-on-prem/installation/redhat-rhel-centos/analytics-pump.md [hunk 11]
```diff
@@ -32,7 +32,7 @@ $ git clone https://github.com/TykTechnologies/tyk-ansible
 $ cd tyk-ansible
 ```
 
-3. Run initalization script to initialize environment
+3. Run initalisation script to initialise environment
```
Comment (andyo-tyk) [inline tyk-docs/content/tyk-on-prem/installation/redhat-rhel-centos/analytics-pump.md]: Typo: _initalisation_

File: tyk-docs/content/tyk-on-prem/installation/redhat-rhel-centos/dashboard.md [hunk 12]
```diff
@@ -32,7 +32,7 @@ $ git clone https://github.com/TykTechnologies/tyk-ansible
 $ cd tyk-ansible
 ```
 
-3. Run initalization script to initialize environment
+3. Run initalisation script to initialise environment
```
Comment (andyo-tyk) [inline tyk-docs/content/tyk-on-prem/installation/redhat-rhel-centos/dashboard.md]: Typo: _initalisation_

File: tyk-docs/content/tyk-on-prem/installation/redhat-rhel-centos/gateway.md [hunk 13]
```diff
@@ -32,7 +32,7 @@ $ git clone https://github.com/TykTechnologies/tyk-ansible
 $ cd tyk-ansible
 ```
 
-3. Run initalization script to initialize environment
+3. Run initalisation script to initialise environment
```
Comment (andyo-tyk) [inline tyk-docs/content/tyk-on-prem/installation/redhat-rhel-centos/gateway.md]: Typo: _initalisation_

File: tyk-docs/content/tyk-on-prem/installation/redhat-rhel-centos/redhat-rhel-centos.md [hunk 14]
```diff
@@ -32,7 +32,7 @@ $ git clone https://github.com/TykTechnologies/tyk-ansible
 $ cd tyk-ansible
 ```
 
-3. Run initalization script to initialize environment
+3. Run initalisation script to initialise environment
```
Comment (andyo-tyk) [inline tyk-docs/content/tyk-on-prem/installation/redhat-rhel-centos/redhat-rhel-centos.md]: Typo: _initalisation_

File: tyk-docs/content/tyk-stack/tyk-developer-portal/enterprise-developer-portal/enterprise-developer-portal.md [hunk 15]
```diff
@@ -10,16 +10,16 @@ url: "tyk-developer-portal/tyk-enterprise-developer-portal"
...
 
-*   Completely customise look and feel of the portal.
-*   Combine multiple APIs into API product and supply them with OpenAPI documentation and tutorials.
-*   Create multiple organization and teams to segment your developer audience.
-*   Create multiple API catalogues to tailor API products and plans visibility to a certain audience.
-*   Integrate with the most popular Identity providers via Dynamic client registration.
-*   Fully control the flow of the developer sign-up and enrollment.
+* Completely customise look and feel of the portal.
+* Combine multiple APIs into API product and supply them with OpenAPI documentation and tutorials.
+* Create multiple organization and teams to segment your developer audience.
```
Comment (andyo-tyk) [inline tyk-docs/content/tyk-stack/tyk-developer-portal/enterprise-developer-portal/enterprise-developer-portal.md]: Missed an _organization_ here -  and should be plural, I think: organisations

File: tyk-docs/content/tyk-stack/tyk-developer-portal/enterprise-developer-portal/enterprise-developer-portal.md [hunk 16]
```diff
@@ -10,16 +10,16 @@ url: "tyk-developer-portal/tyk-enterprise-developer-portal"
...
+* Create multiple API catalogues to tailor API products and plans visibility to a certain audience.
```
Comment (andyo-tyk) [inline tyk-docs/content/tyk-stack/tyk-developer-portal/enterprise-developer-portal/enterprise-developer-portal.md]: Perhaps re-word this, moving "visibility" for readability, such as:
Create multiple API catalogues to tailor visibility of API products and plans to different audiences.

File: tyk-docs/content/tyk-stack/tyk-developer-portal/enterprise-developer-portal/getting-started-with-enterprise-portal/customise-menus.md [hunk 17]
```diff
@@ -27,7 +27,7 @@ Each menu item may:
 - show a dropdown list with possible navigational options:
   {{< img src="img/dashboard/portal-management/enterprise-portal/dropdown-menu-item.png" alt="Dropdown menu item" >}}
 
-Admin users can create additional navigational menus and render them on any page of the live portal. This customization requires changes to a theme and is covered in the [Full customization section]({{< ref "/content/tyk-stack/tyk-developer-portal/enterprise-developer-portal/customise-enterprise-portal/full-customisation/menus-customisation.md" >}}).
+Admin users can create additional navigational menus and render them on any page of the live portal. This customization requires changes to a theme and is covered in the [Full customisation section]({{< ref "/content/tyk-stack/tyk-developer-portal/enterprise-developer-portal/customise-enterprise-portal/full-customisation/menus-customisation.md" >}}).
```
Comment (andyo-tyk) [inline tyk-docs/content/tyk-stack/tyk-developer-portal/enterprise-developer-portal/getting-started-with-enterprise-portal/customise-menus.md]: Missed a _customization_ in the second sentence.

File: tyk-docs/content/tyk-stack/tyk-gateway/install/ce-ansible.md [hunk 18]
```diff
@@ -28,7 +28,7 @@ $ git clone https://github.com/TykTechnologies/tyk-ansible
 $ cd tyk-ansible
 ```
 
-3. Run initalization script to initialize environment
+3. Run initalisation script to initialise environment
```
Comment (andyo-tyk) [inline tyk-docs/content/tyk-stack/tyk-gateway/install/ce-ansible.md]: Typo: initalisation

File: tyk-docs/content/tyk-stack/tyk-gateway/install/ce-debian-ubuntu.md [hunk 19]
```diff
@@ -33,7 +33,7 @@ $ git clone https://github.com/TykTechnologies/tyk-ansible
 $ cd tyk-ansible
 ```
 
-3. Run initalization script to initialize environment
+3. Run initalisation script to initialise environment
```
Comment (andyo-tyk) [inline tyk-docs/content/tyk-stack/tyk-gateway/install/ce-debian-ubuntu.md:36]: Typo: initalisation

File: tyk-docs/content/tyk-stack/tyk-gateway/install/ce-redhat-rhel-centos.md [hunk 20]
```diff
@@ -34,7 +34,7 @@ $ git clone https://github.com/TykTechnologies/tyk-ansible
 $ cd tyk-ansible
 ```
 
-3. Run the initalization script to initialise your environment
+3. Run the initalisation script to initialise your environment
```
Comment (andyo-tyk) [inline tyk-docs/content/tyk-stack/tyk-gateway/install/ce-redhat-rhel-centos.md:37]: Typo: initalisation

**Review by andyo-tyk — CHANGES_REQUESTED** [review]
Awesome effort - just a few that you missed; I think there might be some more that will be visible once this is merged.

File: tyk-docs/content/basic-config-and-security/security/Authentication & Authorization/auth-code-grant-type.md (hunk 1 above)
Comment (marksou) [reply tyk-docs/content/basic-config-and-security/security/Authentication & Authorization/auth-code-grant-type.md:47]: Code format

File: tyk-docs/content/basic-config-and-security/security/Authentication & Authorization/oauth-2-0.md (hunk 3 above)
Comment (marksou) [reply tyk-docs/content/basic-config-and-security/security/Authentication & Authorization/oauth-2-0.md:43]: It is, anything wrapped in `something` will be in code format when compiled in to site

**Review by andyo-tyk — APPROVED** [review]
Thanks for this, Mark - looks great.

**Pull request merged**